File: Kotlin/src/QuickBrownFox.kt

fun main() {
    val antCaser = readLine()?.toInt()

    for (i in 0 until antCaser!!) {
        var base = "abcdefghijklmnopqrstvwxyz"
        val setning = readLine()?.lowercase()

        if (setning != null) {
            for (c in setning) {
                if (c in base) base = base.replace(c.toString(), "")
            }
        }
        when (base) {
            "" -> println("pangram")
            else -> println("missing $base")
        }
    }
}

File: Python/quickBrownFox.py
import sys

antCaser = int(sys.stdin.readline())
for i in range(antCaser):
    base = "abcdefghijklmnopqrstuvwxyz"
    for c in sys.stdin.readline():
        if c.lower() in base:
            base = base.replace(c.lower(), "")
    
    if base == "": print("pangram")
    else: print("missing", base)
